app/queues/common/expiration-request.server.ts

```diff
@@ -37,23 +38,42 @@ const expirationRequestWorker = new Worker(
       logger.info('process DNS record expiration');
       let dnsRecords = await getExpiredDnsRecords();
       await Promise.all(
-        dnsRecords.map(async ({ id, user }) =>
+        dnsRecords.map(async ({ id, subdomain, user }) =>
           Promise.all([
-            // delete records from Route53 and DB
+            // delete DNS records from DB
             deleteDnsRecordById(id),
-            // add notification jobs (assuming deletion went successfully)
+            // add notification jobs
             addNotification({
               emailAddress: user.email,
-              subject: 'DNS record expiration subject',
-              message: 'DNS record expiration message',
+              subject: 'My.Custom.Domain DNS record expired',
+              message: `${user.displayName}, your DNS record with subdomain: ${subdomain} has expired.`,
             }),
           ])
         )
       );
     } catch (err) {
       throw new UnrecoverableError(`Unable to process DNS record expiration: ${err}`);
     }
-    logger.info('TODO: process certificate expiration');
+    try {
```

Comment:
This way, you will also send notification about expired certificate, even if the cert was renewed in the meantime

Reply:
Those certificates remain in the DB, but we are only ever showing the newest one for the user

Reply:
Thanks for your feedback. 
So we only need to notify the user about the newest one?

Reply:
Yes, if they have already renewed it, then there is no point in notifying them about a cert that has been renewed in the meantime, and they can no longer see in the UI (at least IMO)